### DISCUSSION TITLE
style: violates naming scheme for scripts [naming_scripts]

### DISCUSSION THREAD

**Discussion by hongbo-miao** [florianschanda/miss_hit#275] [Q&A] [answered]
I have file at **simulink/main.m**

I got error:

```shell
simulink/main.m: style: violates naming scheme for scripts [naming_scripts]
```

I didn't find any document after searching in this GitHub repo or Google.

How can I have correct file name? Thanks!

**Comment by florianschanda** [comment, marked as answer]
You can adjust the naming scheme using the `naming_scripts` config entry. By default it is `([A-Z]+|[A-Z][a-z]*)(_([A-Z]+|[A-Z][a-z]*|[0-9]+))*` so that would mean you'd want `Main.m`. But please adjust it to whatever regex you'd like! :)

Reference docs: https://florianschanda.github.io/miss_hit/style_checker.html

**Reply by hongbo-miao** [reply]
Thanks @florianschanda ! Missed that part of document. 😃
